subsystems/viamserver/viamserver.go

```diff
@@ -77,8 +77,8 @@ func (s *viamServer) Start(ctx context.Context) error {
 		s.shouldRun = true
 	}
 
-	stdio := agent.NewMatchingLogger(s.logger, false)
-	stderr := agent.NewMatchingLogger(s.logger, true)
+	stdio := agent.NewMatchingLogger(s.logger, false, false)
+	stderr := agent.NewMatchingLogger(s.logger, true, false)
```

Comment:
done in 1190ea58 but if we get the logger.Core().Write() stuff above working, this will change again, so don't look until we've made a design decision

note that if those early logs happen in the pre-online section, they will get missed anyway

Reply:
So this now has the opposite problem. It no longer LOCALLY logs viam-server output at all. I just tested this with a quick build on my own pi.

Local logging needs to always happen (as it'd never be duplicated.) It's only the CLOUD logging where we need to log early startup of viam-server (in case it fails), then stop once it's online and doing its own cloud logging.